client/google-oauth-client/src/main/java/com/wypl/googleoauthclient/exception/GoogleOAuthErrorCode.java

```diff
@@ -0,0 +1,22 @@
+package com.wypl.googleoauthclient.exception;
+
+import com.wypl.common.exception.ServerErrorCode;
+
+import lombok.Getter;
+
+@Getter
+public enum GoogleOAuthErrorCode implements ServerErrorCode {
+	BAD_REQUEST(400, "GOOGLE_OAUTH_001", "권한이 없습니다."),
+	MALFORMED(400, "GOOGLE_OAUTH_002", "오염된 토큰입니다."),
```

Comment:
```suggestion
	MALFORMED(400, "GOOGLE_OAUTH_002", "올바르지 않은 토큰입니다."),
```

이 처럼 수정하면 괜찮지 않을까요?!

Reply:
좋은 의견 감사합니다!
반영했습니다.